packages/auth/src/authClients/ethereum.ts

```diff
@@ -3,7 +3,7 @@ export type EthereumUser = {
 }
 
 export type Ethereum = {
-  login(): Promise<any>
+  login(options: any): Promise<any>
```

Comment:
Can we get stronger typing for the options?

Reply:
Since the client is still under heavy development, is it ok to leave things loose?

I don't want to keep bothering you with PRs to make changes here.

Reply:
Can you export something from the client that can be used here? Then when the client changes we get the new typings automatically

Reply:
(it's ok to say "no")

Reply:
Adding typescript typings is on my todo list. Would it be ok if I add this in next update?

Reply:
Absolutely